ProcessLib/BoundaryCondition/NeumannBoundaryConditionLocalAssembler.h

```diff
@@ -42,30 +44,34 @@ class UniformNeumannBoundaryConditionLocalAssembler final
                   double const t, const GlobalVector& /*x*/,
                   GlobalMatrix& /*K*/, GlobalVector& b) override
     {
-        (void)t;  // TODO time-dependent Neumann BCs
-
         _local_rhs.setZero();
 
         IntegrationMethod integration_method(Base::_integration_order);
         std::size_t const n_integration_points =
             integration_method.getNumberOfPoints();
 
-        for (std::size_t ip(0); ip < n_integration_points; ip++) {
+        SpatialPosition pos;
+        pos.setElementID(id);
+
+        for (std::size_t ip(0); ip < n_integration_points; ip++)
+        {
+            pos.setIntegrationPoint(ip);
             auto const& sm = Base::_shape_matrices[ip];
             auto const& wp = integration_method.getWeightedPoint(ip);
-            _local_rhs.noalias() +=
-                sm.N * _neumann_bc_value * sm.detJ * wp.getWeight();
+            _local_rhs.noalias() += sm.N *
+                                    _neumann_bc_value.getTuple(t, pos).front() *
```

Comment:
I'm against `operator()(...)`, since this method does not have a name at all.
If there should be a method returning the only value, I'd call it `getScalarValue()` or something like that.